prow/plugins/trigger/pr.go

```diff
@@ -83,6 +87,16 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 				return buildAll(c, pr.PullRequest)
 			}
 		}
+		// If a member's PR is labeled "needs-ok-to-test" accidentally (e.g., a maintainer adds it manually by mistake),
+		// just remove it.
+		if pr.Label.Name == needsOkToTest {
```

Comment:
Can we also check that the sender is not the bot, so we can save tokens and make sure we're not going to enter into a bot fighting loop?